internal/Restore-DBFromFilteredArray.ps1

```diff
@@ -371,7 +390,7 @@ Function Restore-DBFromFilteredArray
 					$RestoreDirectory = ((Split-Path $Restore.RelocateFiles.PhysicalFileName) | sort-Object -unique) -join ','
 					$RestoredFile = (Split-Path $Restore.RelocateFiles.PhysicalFileName -Leaf) -join ','
 				}
-				if (!$ScriptOnly)
+				if ($ScriptOnly -eq $false)
```

Comment:
I did try with(!($ScriptOnly)) which worked, but was hard to read so thought I'd just use a few more characters :)